src/System.Runtime.Extensions/tests/EnvironmentExitConsoleApp/EnvironmentExitConsoleApp.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Rather than adding this, can you just use the remote executor app I previously added and we're using in a few places?  My goal for that was that in 90% of these cases we wouldn't need another test app and could just use that with a lambda defining the code to run.